extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -46,32 +46,48 @@ define('ext.wikia.adEngine.video.vastUrlBuilder', [
 		return aspectRatio >= 1 || !isNumeric(aspectRatio) ? adSizes.horizontal : adSizes.vertical;
 	}
 
-	function buildAdUnit(slotParams) {
-		var adUnitBuilder = adContext.getContext().opts.megaAdUnitBuilderEnabled ? megaAdUnitBuilder : regularAdUnitBuilder;
```

Comment:
Right now it will be totally outside Disaster recovery, are we ok with that? Please note that in ticket